packages/engine/src/avatar/AvatarInputSystem.ts

```diff
@@ -270,10 +270,12 @@ const execute = () => {
           reDoubleClickTimer = 0
         }
       } else {
+        const avatarClicked = isAvatarClicked()
```

Comment:
we should compute this as `const avatarClicked = buttons.PrimaryClick?.up && isAvatarClicked()` to avoid running the physics raycast when `buttons.PrimaryClick?.up` is not true

Reply:
Yup it's done